docs/influxdb.rst

```diff
@@ -61,17 +61,28 @@ To use our connector from your code, import the connector:
     }
     write_api.write(bucket, org, [data])
 
-InfluxDB connector in gather_job_data.py
+InfluxDB connector in gather_data.py
 ----------------------------------------
 
+Job data:
+
 -   **Measurement:** failure type (or "successful jobs");
--   **Tags:** job_name, job_id, run_id, branch, commit_sha, platform, status,
+-   **Tags:** job_name, job_id, run_id, branch, commit_sha, platform, conclusion,
     runner_label;
 -   **Fields:** time the job was in queue, time the job was in progress;
```

Comment:
'time the job was in queue' is a timestamp or a timedelta? same with the other value.

Reply:
fixed